MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,14 +20,23 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onFeedError: (() -> Void)?
+	var onHideFeedError: (() -> Void)?
```

Comment:
ViewModel can control hiding and showing the error with one `Observer` of `String?`. If it is called with a non-nil value then errorView is shown with the message passed and if it is called with a nil value the error view is hidden.
```suggestion
	var onErrorStateChange: Observer<String?>?
```